File: Mobile/app/src/main/java/com/example/myapplication/Model/Pizza.java
package com.example.myapplication.Model;

import com.facebook.stetho.json.annotation.JsonValue;
import com.google.gson.annotations.SerializedName;

public class Pizza
{
    private int id;

    @SerializedName("sorte") 
    private String sorte;



    @SerializedName("prixPetite")
    private float prixPetite;

    @SerializedName("prixMoyenne")
    private float prixMoyenne;

    @SerializedName("prixGrande")
    private float prixGrande;

    @SerializedName("idImage")
    private int image;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getSorte() {
        return sorte;
    }

    public void setSorte(String sorte) {
        this.sorte = sorte;
    }

    public int getImage() {
        return image;
    }

    public void setImage(int image) {
        this.image = image;
    }

    public float getPrixPetite() {
        return prixPetite;
    }

    public void setPrixPetite(float prixPetite) {
        this.prixPetite = prixPetite;
    }

    public float getPrixMoyenne() {
        return prixMoyenne;
    }

    public void setPrixMoyenne(float prixMoyenne) {
        this.prixMoyenne = prixMoyenne;
    }

    public float getPrixGrande() {
        return prixGrande;
    }

    public void setPrixGrande(float prixGrande) {
        this.prixGrande = prixGrande;
    }
}

File: Backend/BackendFinalBossCegep/BackendFinalBossCegep/Model/Pizza.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace BackendFinalBossCegep.Model
{
    [Table("pizza")]
    public class Pizza
    {
        [Column("id")]
        public int Id { get; set; }
        [Column("sorte")]
        public string Sorte { get; set; }
        
        [Column("prix_petite")]
        public float PrixPetite { get; set; }
        [Column("prix_moyenne")] 
        public float PrixMoyenne { get; set; }
        [Column("prix_grande")]
        public float PrixGrande { get; set; }

        [Column("image")]
        public int IdImage { get; set; }
    }
}
